conda/base/constants.py

```diff
@@ -185,11 +185,7 @@
     "#",
 }
 # Includes disallowed characters from constructor
-PREFIX_NAME_DISALLOWED_CHARS_WIN = {
-    "/",
-    " ",
-    ":",
-    "#",
+PREFIX_NAME_DISCOURAGED_CHARS_WIN = {
```

Comment:
It could even make sense to not add this symbol here or at least prefix it and underscore since it its TBD how this will be handled.